plugins/kubernetes/test/models/kubernetes/deploy_executor_test.rb

```diff
@@ -960,6 +960,34 @@ def expect_logs(logs = "LOG-1")
     end
   end
 
+  describe "#sum_event_group" do
```

Comment:
that's kinda cheating, but whatever :D

Reply:
Haha - yer sorry about that one chief.
I was going to try & figure out how to set up the right conditions for the test of an existing public method, but when I saw the `#show_failure_cause` test is already testing a private method thought I could just add this test in to get things moving again.
(Thanks for the merge though - hopefully we'll be able to see what the *real* problem is now.)